.ocm/branch-info.yaml

```
@@ -0,0 +1,5 @@
+release-branch-template: release-v$major.$minor # e.g. release-v1.0
+branch-policy:
+  significant-part: minor # major, minor, patch
+  supported-versions-count: 2
+  release-cadence: 2w # d (days), w (weeks), y | yr (years)
```

A feedback from @dimitar-kostadinov :

We support only the latest minor release. And we don't have a fixed release cadence for this extension.

```suggestion
  supported-versions-count: 1
  release-cadence: null # d (days), w (weeks), y | yr (years)
```